minecraft_servers/tariamc/manifest.json

```diff
@@ -0,0 +1,41 @@
+{
+    "server_name": "tariamc",
+    "nice_name": "TariaMC",
+    "direct_ip": "tariamc.net",
+    "server_wildcards": [
+        "%.tariamc.net"
+    ],
+    "supported_languages": [
+        "en",
+        "de"
+    ],
+    "gamemodes": {
+        "molecraft": {
+            "name": "MoleCraft",
+            "color": "#5beb34"
+        },
+        "boxxed": {
+            "name": "Boxxed",
+            "color": "#3471eb"
+        },
+        "starblock": {
+            "name": "StarBlock",
+            "color": "#FFE81F"
+        }
+    },
+    "social": {
+        "web": "https://tariamc.net/",
+        "twitter": "TariaMCnet",
+        "discord": "https://discord.com/invite/uAc9C9c2JD",
+        "teamspeak": "tariamc.net"
+    },
+    "brand": {
+        "text": "#FFFF",
+        "background": "#306fdb",
+        "primary": "#306fdb"
+    },
+    "discord": {
+        "server_id": 952950096843120690,
+        "rename_to_minecraft_name": true
```

Comment:
Please use `false`, as this is the default value. We will accept `true` as a value here only for verified partners.